helmfile/values/podsecuritypolicies/common/rook-ceph.yaml.gotmpl

```diff
@@ -0,0 +1,226 @@
+{{- if .Values.rookCeph.enabled }}
+constraints:
+  rook-ceph:
+    rook-ceph-operator:
```

Comment:
You can drop all rook-ceph for services, as they are under the rook-ceph namespace
```suggestion
    operator:
```